mcs/class/System.ServiceModel/System.ServiceModel.Channels.Http/HttpListenerManager.cs

```diff
@@ -48,24 +48,29 @@ protected HttpListenerManager ()
 			Entries = new List<HttpChannelListenerEntry> ();
 		}
 
-		public List<HttpChannelListenerEntry> Entries { get; private set; }
+		protected List<HttpChannelListenerEntry> Entries { get; private set; }
```

Comment:
It can't be private as it's used in a subclass (HttpStandaloneListenerManager). Since this is an internal class only our code can subclass it and I've verified we don't need locking there (it only accesses .Count).

FWIW I don't like the way it is too and had the same concerns, but didn't want to change too much code given that we hope to replace this with referencesource ServiceModel in the future anyway.